etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamSenderHelper.java

```diff
@@ -55,23 +55,23 @@ protected Optional<String> sendToReportStream(
         logger.logInfo("{} successfully sent to ReportStream", messageType);
         metadata.put(fhirResourceId, EtorMetadataStep.SENT_TO_REPORT_STREAM);
 
-        Optional<String> sentSubmissionId = getSubmissionId(rsResponseBody);
-        if (sentSubmissionId.isEmpty()) {
-            logger.logError("Unable to retrieve sentSubmissionId from ReportStream response");
+        Optional<String> outboundReportId = getReportId(rsResponseBody);
+        if (outboundReportId.isEmpty()) {
+            logger.logError("Unable to retrieve ReportId from ReportStream response");
         } else {
-            logger.logInfo("ReportStream response's sentSubmissionId={}", sentSubmissionId);
+            logger.logInfo("ReportStream response's ReportId={}", outboundReportId);
         }
 
-        return sentSubmissionId;
+        return outboundReportId;
     }
 
-    protected Optional<String> getSubmissionId(String rsResponseBody) {
+    protected Optional<String> getReportId(String rsResponseBody) {
         try {
             Map<String, Object> rsResponse =
                     formatter.convertJsonToObject(rsResponseBody, new TypeReference<>() {});
-            return Optional.ofNullable(rsResponse.get("submissionId").toString());
+            return Optional.ofNullable(rsResponse.get("reportId").toString());
```

Comment:
I may be misunderstanding this, but it looks like it's parsing a field from a ReportStream response, and we're now looking for a different field than we were before. Is that the intention/are both `submissionId` and `reportId` real fields and we're switching which we're using?

Reply:
Exactly.  Both `submissionId` and `reportId` are fields in a real RS response.  We are changing what we're using.